beetsplug/vocadb.py

```diff
@@ -162,11 +160,27 @@ class AlbumFindResultDict(FindResultDict):
     items: list[AlbumDict]
 
 
+class ConfigDict(TypedDict):
+    prefer_romaji: bool
+    translated_lyrics: bool
+    include_featured_album_artists: bool
+    va_string: str
+    max_results: int
+
+
 class VocaDBPlugin(BeetsPlugin):
 
+    default_config: ConfigDict = {
+        "prefer_romaji": False,
+        "translated_lyrics": False,
+        "include_featured_album_artists": False,
+        "va_string": "Various artists",
+        "max_results": 5,
```

Comment:
```suggestion
        "max_results": "5",
```

I am getting error:
`configuration error: vocadb.max_results: must be a string`
when I don't set `max_results` in beets config.